src/server/builds/node.js

```diff
@@ -210,7 +210,7 @@ AlgoliaSearchNodeJS.prototype._request = function request(rawUrl, opts) {
 
       abort();
       clearTimeout(timeoutId);
-      reject(new errors.NodeNetwork(err.message, err));
+      reject(new errors.Network(err.message, err));
```

Comment:
Yes it is. I just made a new commit where we always try all hosts when there's a network error before switching to JSONP.

Trying to switch to JSONP early was made by thinking the only network error that could happen was blocked XHR. While it can be a DNS failure on the browser, we should retry using XHR.